bin/seed-golfathlete-db.js

```diff
@@ -0,0 +1,70 @@
+#! /usr/bin/env node
+
+'use strict'
+
+const async = require('async')
+const Athlete = require('../src/models/athlete')
+
+const mongoose = require('mongoose')
+mongoose.set('useCreateIndex', true)
+mongoose.connect('mongodb://localhost:27017/golfAthleteTest', { useNewUrlParser: true })
+mongoose.Promise = global.Promise
+const db = mongoose.connection
+db.on('error', console.error.bind(console, 'MongoDB connection error:'))
+
+const athletes = []
+
+function createAthlete (lastName, firstName, ghinNumber, cachedGhinIndex, cachedGhinIndexDate, driverClubHeadSpeed, dob, address, description, cb) {
+  const now = Date.now()
+
+  const athlete = new Athlete({
+    lastName: lastName,
+    firstName: firstName,
+    ghinNumber: ghinNumber,
+    cachedGhinIndex: cachedGhinIndex,
+    cachedGhinIndexDate: cachedGhinIndexDate,
+    driverClubHeadSpeed: driverClubHeadSpeed,
+    dob: dob,
+    address: address,
+    description: description,
+    createdAt: now,
+    updatedAt: now
+  })
+
+  athlete.save(function (err) {
+    if (err) {
+      cb(err, null)
+      return
+    }
+    console.log('New Athlete: ' + athlete)
```

Comment:
maybe use `debug` if u already have it 😉

Reply:
maybe all `console.log` should/would be replaced by `debug`

Reply:
Yes. Just a habit. I was planning to remove console.log lines and missed do that. For that matter, I still need to implement better logging throughout the code. Debug where it is needed and error logs as necessary.